source/_components/sensor.dnsip.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: page
+title: "DNS IP"
+description: "Instructions on how to integrate a DNS IP sensor into Home Assistant."
+date: 2017-02-25 11:05
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Sensor
+ha_iot_class: "Cloud Polling"
+ha_release: 0.40
+---
+
+
+The `dnsip` sensor will expose an IP address, fetched via DNS resolution, as its value. There are two operational modes:
```

Comment:
Isn't it just that there are two use cases? The modes really use the same code and logic just with a different hostname. I think it may be better to just cover the use cases with more explanation near the examples.